modules/nlohmann_json/3.11.3.bcr.1/patches/add_singleheader_target.patch

```diff
@@ -0,0 +1,15 @@
+--- BUILD.bazel
++++ BUILD.bazel
+@@ -51,3 +51,12 @@
+     visibility = ["//visibility:public"],
+     alwayslink = True,
+ )
++
++cc_library(
++    name = "singleheader/json",
```

Comment:
probably best to not have a slash in the name here?

Reply:
See reasoning why I think it would be a neat idea here in https://github.com/chipsalliance/verible/pull/2268

But making it a dash (`:singleheader-json`) would also be fine by me.